app/controllers/donations_controller.rb

```diff
@@ -13,9 +13,11 @@ def index
                                      .order(created_at: :desc)
                                      .where(issued_at: date_range)
                                      .class_filter(filter_params)
+    @paginated_donations = @donations.page(params[:page])
     # Are these going to be inefficient with large datasets?
     # Using the @donations allows drilling down instead of always starting with the total dataset
     @donations_quantity = @donations.collect(&:total_quantity).sum
+    @paginated_donations_quantity = @paginated_donations.collect(&:total_quantity).sum
```

Comment:
Would
```suggestion
    @paginated_donations_quantity = @paginated_donations.sum(:total_quantity)
```
work? (Not rhetorical, I'm honestly not sure if it would or not)

Reply:
I dont see a reason why it wouldn't I was just using the established pattern already present in this controller. (I was trying not to question existing code too much and keep the changes restricted to what is relevant)

Happy to change it but if I do we should probably do it in both places. (ie also in `@donation_quanity`)

Reply:
Persisting existing patterns is a perfectly valid reason :)